op-batcher/batcher/service.go

```diff
@@ -428,6 +428,14 @@ var _ cliapp.Lifecycle = (*BatcherService)(nil)
 
 // Driver returns the handler on the batch-submitter driver element,
 // to start/stop/restart the batch-submission work, for use in testing.
-func (bs *BatcherService) Driver() rpc.BatcherDriver {
+func (bs *BatcherService) Driver() *BatchSubmitter {
 	return bs.driver
 }
+
+// NewTestDriver returns a handler for the batch-submitter driver element, to start/stop/restart
+// the batch-submission work, for use only in testing.
+func (bs *BatcherService) NewTestDriver() *TestBatchSubmitter {
```

Comment:
nit, to be consistent with `Driver`
```suggestion
func (bs *BatcherService) TestDriver() *TestBatchSubmitter {
```